### ISSUE TITLE
Error while installing previous core versions

### ISSUE THREAD

**Issue by mateumiralles** [dappnode/DNP_CORE#180]
Doing a downgrade to a previous version of core its not allowed. That way is it mandatory to install every core package independently to do rollback to production version for testing purposes. 
- Example of the error:
`DAppNode Package is not compatible: dappGet could not resolve request core.dnp.dappnode.eth@/ipfs/QmUV19ocNdgpSkWxrjHC7KDDs6GtW2eQioqChqp7ttbFiz, error on aggregate stage: Aggregated versions of installed package ipfs.dnp.dappnode.eth cause a downgrade from 0.2.23. Having a future development version could be the cause of this error.
`

**Comment by alexpeterson91** [comment]
Technically yes. Really you need to just always have core component package versions <= to the DNP_CORE release you're downgrading to's dependencies which can be found in the releases manifest file, i.e. 

Depending on what dependencies are downgraded and what release it is (a major release like when we added stakers ui or added rollups ui or updated bind and started the ip migration etc any major breaking release it's absolutely necessary to downgrade everything. 

(Note the core component packages (and many other non core packages) have minimum required dappnode versions which are dappmanager versions and may further complicate downgrading core on a node for test purposes but if you're only rolling back a version or so that's not major you'll have far fewer dependency issues downgrading core. 

**tl/dr**
It's dependency issues you're running into Yu can find which packages (it may not even be a core package some packages like the main stakers chains clients have minimum dappmanager versions because the features that enable them aren't available unyil that dappmanager release) that's preventing a core downgrade by checking both the core component package manifests and the core package manifest. Non core packages that have minimum versions are again not dependent on dnp_core versions but rather dappmanager versions.
